sources/logdog/Cargo.toml

```diff
@@ -8,10 +8,15 @@ publish = false
 # Don't rebuild crate just because of changes to README.
 exclude = ["README.md"]
 
+[features]
+fips = ["rustls/fips", "aws-lc-rs/fips"]
+
 [dependencies]
+aws-lc-rs = { workspace = true, features = ["bindgen"] }
 flate2 = { workspace = true, features = ["default"] }
 glob.workspace = true
 reqwest = { workspace = true, features = ["blocking", "rustls-tls-native-roots"] }
+rustls.workspace = true
```

Comment:
`logdog` is another one where we don't really need TLS, right?

Reply:
Actually, `logdog` "supports" https:

https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/62450eb4c60a52ecbcb60b4abfdfea63e63e5ebf/sources/logdog/src/log_request.rs#L153